mappings/net/minecraft/server/world/ServerEntityManager.mapping

```diff
@@ -17,9 +17,9 @@ CLASS net/minecraft/class_5579 net/minecraft/server/world/ServerEntityManager
 	METHOD method_31843 removeEntity (Lnet/minecraft/class_5568;)V
 		ARG 1 entity
 	METHOD method_31845 getDebugString ()Ljava/lang/String;
-	METHOD method_31847 (Lnet/minecraft/class_5568;)V
+	METHOD method_31847 loadEntity (Lnet/minecraft/class_5568;)V
 		ARG 1 entity
-	METHOD method_31850 (Lnet/minecraft/class_5568;)V
+	METHOD method_31850 unloadEntity (Lnet/minecraft/class_5568;)V
```

Comment:
Can't agree on these. They are not really load/unload any more; they are now like start/stop tracking/ticking, and additionally they now have a callback for entity creation/removal (permanent)

See https://github.com/FabricMC/yarn/pull/1967/files#diff-f7ca4a3bef6f22a6a76332ff1ec5cf595cf7b1f658c65bf46ad4e9a06f4815ccR102